### ISSUE TITLE
[Bug] GP tooltip values seem incorrect

### ISSUE THREAD

**Issue by mykkenny** [Alumian/CEPGP-Retail#78]
**The Error**
Rare MH: https://i.imgur.com/hR5KSkn.png
Epic OH: https://i.imgur.com/NIxK2Jd.png

**Describe the bug**
They have the same value. In the first place the formula is meant to take into account rarity, and in the second MH modifier is 1.5 and OH is 0.5. For reference our formula looks like:

`4.83x(1^((ilvl/26)+(rarity-4)) x slot) x10`

Can also see that a rare pare of pants that I have from ST dungeon are the same value as epic legs from MC?

**To Reproduce**
Look at any piece of gear and check the GP value against what it should be according to the formula

**Comment by skombag** [comment]
its not a bug its following the formula, default formula ignores rarity and itemlevel, because it uses (1^ ((ilvl/26)+(rarity-4)) x slot) , so 1 to any power is 1, for it to matter you need to use 2 instead. and you have to adjust initial value of 4,83 to match your desiared GP range because now you will have much bigger values.

i personally use inflated values for both EP and GP to account for the rounding up problem some versions ago:
0.483 x (2^ ((ilvl/26)+(rarity-4)) x slot) x 100

**Comment by skombag** [comment]
got a new error on clicking on item posted in chat.

Date: 2019-10-20 19:12:14
ID: 2
Error occured in: Global
Count: 1
Message: ..\AddOns\CEPGP\Utility.lua line 125:
   Usage: Item:CreateFromItemID(itemID)
Debug:
   [C]: error()
   ..\FrameXML\ObjectAPI\Item.lua:44: CreateFromItemID()
   CEPGP\Utility.lua:125: CEPGP_calcGP()
   CEPGP\Utility.lua:330:
      CEPGP\Utility.lua:326
   [C]: ?
   [C]: ChatFrame_OnHyperlinkShow()
   [string "*:OnHyperlinkClick"]:1:
      [string "*:OnHyperlinkClick"]:1
Locals:
(*temporary) = "Usage: Item:CreateFromItemID(itemID)"

AddOns:
  Swatter, v8.2.6377 (SwimmingSeadragon)
  D4KiRHealerProtection, v3.1 Release
  AceGUI30SelectablePanel, vv1.0-alpha2
  AdvancedInterfaceOptions, v1.3.3
  AngryAssignments, vv1.11.10
  AtlasLootClassic, vv1.3.1
  AtlasLootClassicData, vv1.3.1
  AtlasLootClassicDungeonsAndRaids, vv1.3.1
  AucAdvanced, v8.2.6430 (SwimmingSeadragon)
  AucFilterBasic, v8.2.6364 (SwimmingSeadragon)
  AucStatHistogram, v8.2.6366 (SwimmingSeadragon)
  AucStatiLevel, v8.2.6370 (SwimmingSeadragon)
  AucStatPurchased, v8.2.6367 (SwimmingSeadragon)
  AucStatSimple, v8.2.6399 (SwimmingSeadragon)
  AucStatStdDev, v8.2.6369 (SwimmingSeadragon)
  AucUtilFixAH, v8.2.6371 (SwimmingSeadragon)
  AutoBarClassic, v1.13.2.06
  Bagnon, v8.2.16
  BankStack, vv11302.0
  BetterAlign, v
  CallbackHandler10, v
  CastCursor, v1.4
  ccc, v0.3.12
  CEPGP, v
  ClassicAuraDurations, v
  ClassicCastbars, vv1.1.1
  ClassicCastbarsOptions, v
  ClassicGuildBank, v1.2
  ClassicLFG, v1.1.3
  ClassIcons, v0.1
  ClassTrainerPlus, v0.6-beta
  ClickLinks, v1.0.8
  Clique, vv80000-1.0.0
  DBMCore, v1.13.16
  DBMDefaultSkin, v
  DBMStatusBarTimers, v
  Details, v
  DetailsTinyThreat, v
  ExtendedCharacterStats, v1.5
  FiveSecondRule, v1.14
  GatherLite, v1.3.3-classic
  GigaAutoResourcesTracker, v0.0.1
  GTFO, v4.51
  GuildRosterManager, v1.76
  HealComm, v
  InFlightLoad, v1.13.003
  Informant, v8.2.6374 (SwimmingSeadragon)
  ISBoxer, v1.0
  ItemRack, v
  ItemRackOptions, v
  LeatrixMaps, v1.13.35
  LeatrixPlus, v1.13.35
  LibSharedMedia30, v3.0-113
  LibStub, v
  LoseControl, v1.00
  mailoutbox, v1.2.0.release
  MikScrollingBattleText, v5.7.150
  oGlow, v2.2.7
  OmniCC, v8.2.5
  oRA3, vv2-classic-4-g4cd51fb
  PixelPerfectAlign, vv1.07.03
  Plater, v
  Prat30, v3.7.36
  Prat30Libraries, v
  Quartz, v3.5.2
  QuestFrameFixer, v1.1.1
  Questie, v4.3.0 0 37acad5
  RealMobHealth, v2.13
  RollTrackerClassic, v1.51
  SaySapped, v2019-08-27
  SilverDragon, vv11302.0
  SilverDragonClassic, vv11302.0
  SkilletClassic, v1.00-beta7
  SlideBar, v8.2.6375 (SwimmingSeadragon)
  Spy, v3.6.9
  Stubby, v8.2.6376 (SwimmingSeadragon)
  TinyTooltip, v8.2.1
  TitanClassic, v1.1.1.11302
  TitanClassicAmmo, v1.1.1.11302
  TitanClassicBag, v1.1.1.11302
  TitanClassicClock, v1.1.1.11302
  TitanClassicGold, v1.1.1.11302
  TitanClassicLocation, v1.1.1.11302
  TitanClassicLootType, v1.1.1.11302
  TitanClassicPerformance, v1.1.1.11302
  TitanClassicRepair, v1.1.1.11302
  TitanClassicVolume, v1.1.1.11302
  TitanClassicXP, v1.1.1.11302
  WeakAuras, v2.15.4
  WeakAurasOptions, v2.15.4
  WoWScribe, v8.0
  WoWScribeDataBroker, v1.0
  XLoot, v8.1-1
  XLootFrame, v8.1-1
  XLootGroup, v8.1-1
  XLootMaster, v8.1-1
  XLootMonitor, v8.1-1
  BlizRuntimeLib_enUS v1.13.2.11302 <none>
  (ck=aab)

**Comment by Alumian** [comment]
Yep Skoms got it. Just change that value in your GP formula and you'll be good to go. But yeah with that error you posted Skom, it was a rushed feature because after the initial implementation of it I noticed a bug which needed to be hotfixed which is why it was prematurely released. I'll keep looking into this but I'm going to close this ticket for the time being since the original issue has been resolved.
